cmd/clef/main.go

```diff
@@ -634,6 +634,19 @@ func signer(c *cli.Context) error {
 		log.Info("Performing UI test")
 		go testExternalUI(apiImpl)
 	}
+
+		// list known accounts to console on startup
+		ui_server := core.NewUIServerAPI(apiImpl)
+		accs, err := ui_server.ListAccounts(c.Context)
+		if err != nil {
+			return err
+		}
```

Comment:
This is not correct. We already instantiate `core.NewUIServerAPI(apiImpl)` at line `569`. If we want to maintain access to it, we should just keep it around instead of instantiating a second one here. 

Now ,beside that... At this point, we already have a `ui`. It migth be a console UI, but it might also be a `stdioui`-ui, i.e.: a UI which uses json-rpc communication over standard-input and standard-output. A proper external UI would use that approach. 

That means that we cannot "just print" stuff on `stdout`. If we need to show information to the user, the idiomatic way of doing it is to tell our `ui` to do so. The main way of doing that is `ui.ShowInfo()` -- that will work both for console users and remote-ui users. 

If we want to display it _only_ in the console, then we can either ensure to print on `stderr` (`fmt.Fprintf(os.Stderr, "...`), or show the info using logging - which is already configured to not disrupt the `stdout`. 

Anyway, I'm not sure I agree with that readme-note any more. The UI can just ask the server about the accounts. If the ui-server is used, as opposed to the _external_ api, then it should not necessitate explicit user approval, so be a very quick operation.

Reply:
ah ok i see what you mean. Sorry - I think I've fixed this now, but if it is not a desired feature then no worries.

Reply:
I think that what would be more idiomatic, would be if the `cliui` itself does this. So instead of `clef` spitting out the accounts on the console, the ui would ask for the accounts, and then show them to the user. 
That's the job of the ui, not the job of the backend. You could hook into the `OnSignerStartup`, and make that call trigger the UI into doing an account listing.

Reply:
In order to do that, you will need to 'capture' the backend that we currently ignore
```golang
func (ui *CommandlineUI) RegisterUIServer(api *UIServerAPI) {
	// noop
}
```
and then extend the method here
```golang
func (ui *CommandlineUI) OnSignerStartup(info StartupInfo) {
	fmt.Printf("------- Signer info -------\n")
	for k, v := range info.Info {
		fmt.Printf("* %v : %v\n", k, v)
	}
}
```